docs/sdks/getting-started.md

```diff
@@ -4,29 +4,42 @@ sidebar_label: Getting Started
 slug: /sdks/getting-started
 ---
 
-## Configuration Options
-Statsig offers 20+ SDKs, with both client-side and server-side SDK options. At their core, Statsig’s SDKs enable two key functions: 
-1. **Targeting & Assignment-** Assigning users to an experiment variant or targeting new feature roll-outs based on a user’s attributes. Target with any user or environment-level attribute you log.  
-2. **Logging-** Log events for user or system-level actions, which are translated into computed Metrics for analysis within your Statsig Console.
+Statsig offers a wide range of SDKs, supporting over **30 platforms** to help you integrate experimentation, feature flagging, and logging functionality directly into your product, whether it’s a web, mobile, or server-side application.
 
-Our web (javascript, react) SDKs also support [autocaptured events for web analytics](/webanalytics/overview), and [session replay](/session-replay/overview).
+Statsig’s SDKs enable two core functions:
 
-You can [learn more here](/sdks/client-vs-server) about the differences between client and server-side SDKs.
+1. **Targeting & Assignment**: Dynamically assign users to experiment variants or control feature rollouts based on their attributes. You can target users by any **user or environment-level attribute** that you log, such as location, device type, or user behavior.
+  
+2. **Logging Events**: Log events for user actions or system behaviors to analyze feature performance or experiment outcomes. These events are used to compute metrics, which are accessible through the **Statsig Console** for in-depth analysis.
 
-### Client SDKs
-- [Javascript](/client/javascript-sdk)
+For web platforms, such as JavaScript and React, Statsig also supports **[autocaptured events for web analytics](/webanalytics/overview)** and **[session replay](/session-replay/overview)** to give you insights into user behavior without manual event tracking.
+
+To learn more about the differences between **client-side** and **server-side** SDKs, you can refer to our [Client vs Server SDK Overview](/sdks/client-vs-server).
+
+---
```

Comment:
nextjs is one of the most popular - its both client and server, so it doesnt really fit this framing.  we need to make sure it is listed prominently